jupyterhub-k8s-hub.yaml

```diff
@@ -73,6 +73,86 @@ subpackages:
             readlink -v /usr/local/etc/jupyterhub/jupyterhub_config.py
             readlink -v /usr/local/etc/jupyterhub/z2jh.py
 
+  - name: jupyterhub-k8s-hub-image-awaiter
+    description: Checker, if all pods are ready in jupyterhub-k8s-hub deamonset.
+    pipeline:
+      - uses: go/build
+        with:
+          ldflags: |
+            -w
+            -s
```

Comment:
Its equivalent of static packages if i am not wrong, are we sure we have to use it?

Reply:
after looking into it furhter it strips certien symbols but we already do that with strip later in the pipeline